db/schema.rb

```diff
@@ -45,7 +45,7 @@
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
     t.string "description"
-    t.boolean "adopted?", default: true
+    t.string "adopted?", default: "Adoptable"
```

Comment:
Now we can edit the adoption status to multiple things instead of two with the boolean it was before.